conf/messages.en

```diff
@@ -80,15 +80,27 @@ selectApplicationType.previousCommodity = Renew a previous commodity code ruling
 selectApplicationType.checkYourAnswersLabel = Select application type
 selectApplicationType.error.required = Select application type
 
-registerBusinessRepresenting.title = registerBusinessRepresenting
-registerBusinessRepresenting.heading = registerBusinessRepresenting
-registerBusinessRepresenting.field1 = Field 1
-registerBusinessRepresenting.field2 = Field 2
-registerBusinessRepresenting.checkYourAnswersLabel = registerBusinessRepresenting
-registerBusinessRepresenting.error.field1.required = Enter field1
-registerBusinessRepresenting.error.field2.required = Enter field2
-registerBusinessRepresenting.error.field1.length = field1 must be 100 characters or less
-registerBusinessRepresenting.error.field2.length = field2 must be 100 characters or less
+registerBusinessRepresenting.title = What are the registered details of the business or organisation you're representing?
+registerBusinessRepresenting.heading = What are the registered details of the business or organisation you're representing?
+registerBusinessRepresenting.eoriNumber = EORI number, example: GB123456789000
+registerBusinessRepresenting.businessName = Business or organisation name
+registerBusinessRepresenting.addressLine1 = Address line 1
+registerBusinessRepresenting.town = Town or city
+registerBusinessRepresenting.postCode = Postcode
+registerBusinessRepresenting.country = Country
+registerBusinessRepresenting.checkYourAnswersLabel =  What are the registered details of the business or organisation you're representing?
+registerBusinessRepresenting.error.eoriNumber.required = Enter their EORI number
+registerBusinessRepresenting.error.eoriNumber.format = Enter their EORI number in the correct format 
+registerBusinessRepresenting.error.businessName.required = Enter their business or organisation name
+registerBusinessRepresenting.error.addressLine1.required = Enter address line 1
```

Comment:
I would remove the suffix `1` in both the key and the message

Reply:
Just to clarify, all this info is in the prototypes. I agree with you but it could be worth to mention to Aimee or Steve